### ISSUE TITLE
Compatibility with Create's railway system

### ISSUE THREAD

**Issue by thisisthedogtor** [jonafanho/Minecraft-Transit-Railway#752]
I often play Minecraft with the Create mod, which adds its own railway system. Unfortunately, players are very limited when it comes to certain mechanics (e.g. train arrival displays and on-board displays). Thus, I'd like to suggest a compatibility with the Create mod's railway system to allow players to use MTR trains on Create mod tracks, and make the passenger displays compatible with Create mod stations.

**Comment by Ultimate-USA-YT** [comment]
Not shure this will work due to the Many Differences between MTR And Create

**Comment by kettle-7** [comment]
Yeah this would require insane amounts of effort for very little actual improvement to the MTR experience.

**Comment by cvrunmin** [comment]
+1 for the interaction with Create's railway system. However given that the massive difference between the railway systems of both mods, it would be very difficult to allow MTR's train to ride on Create's track and Create's train contraption on MTR's rail. I suggest adding a block connecting to create:station block that register a selected tracks linked by the station block as a pseudo "platform" in the dashboard. Pseudo platform can be added into routes and allow such route to be displayed in route map. Using pseudo platforms MTR's cannot manage the train schedule on such route and Create will takeover it, including train arrival time (function of PIDS would be very limited tho, to be admitted).
